base/sparse.jl

```diff
@@ -45,4 +44,37 @@ if Base.USE_GPL_LIBS
     include("sparse/spqr.jl")
 end
 
+# point users to SuiteSparse
+const SUITESPARSE_END_ERR_SRING = string(" has been moved to the package SuiteSparse.jl.\n",
+          "Run Pkg.add(\"SuiteSparse\") to install SuiteSparse on Julia v0.5-")
+
+"""
+    etree(A[, post])
+Compute the elimination tree of a symmetric sparse matrix `A` from `triu(A)`
+and, optionally, its post-ordering permutation.
+
+Note: This function has been moved to the SuiteSparse.jl package.
+"""
+etree{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, postorder::Bool) =
+    error("etree(A[, post])", SUITESPARSE_END_ERR_SRING)
+etree(A::SparseMatrixCSC) = etree(A, false)
+
+ereach{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, k::Integer, parent::Vector{Ti}) =
+    error("ereach(A, k, parent)", SUITESPARSE_END_ERR_SRING)
+
+csc_permute{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, pinv::Vector{Ti}, q::Vector{Ti})=
+    error("csc_permute(A, pinv, q)", SUITESPARSE_END_ERR_SRING)
+
+"""
+    symperm(A, p)
+Return the symmetric permutation of `A`, which is `A[p,p]`. `A` should be
+symmetric, sparse, and only contain nonzeros in the upper triangular part of the
+matrix is stored. This algorithm ignores the lower triangular part of the
+matrix. Only the upper triangular part of the result is returned.
+
+Note: This function has been moved to the SuiteSparse.jl package.
+"""
+symperm{Tv,Ti}(A::SparseMatrixCSC{Tv,Ti}, pinv::Vector{Ti}) =
+    error("symperm(A, pinv)", SuiteSparse_END_ERR_SRING)
```

Comment:
The errors and docstrings look fine locally and these methods are probably not used at all as much as the Pkg stuff so this might not need to be tested.